pkg/project/admission/requestlimit/admission.go

```diff
@@ -61,6 +61,9 @@ var _ = oadmission.Validator(&projectRequestLimit{})
 
 // Admit ensures that only a configured number of projects can be requested by a particular user.
 func (o *projectRequestLimit) Admit(a admission.Attributes) (err error) {
+	if o.config == nil {
```

Comment:
hmm, we could have them implement a new interface that says whether they want to be excluded